source/fundamentals/crud/write-operations/delete.txt

```diff
@@ -0,0 +1,210 @@
+   :dedent:
+   :start-after: start-model
+   :end-before: end-model
+
+This collection is from the :atlas:`sample datasets </sample-data>` provided
+by Atlas. See the :ref:`<csharp-quickstart>` to learn how to create a free MongoDB cluster
+and load this sample data.
+
+Delete Operations
+-----------------
+
+Use **delete operations** to remove data from MongoDB. Delete operations
+use the ``DeleteOne()`` method, which deletes *the first document* that
+matches your query filter, and the ``DeleteMany()`` method, which
+deletes *all documents* that match the filter.
+
+Delete a Document
+~~~~~~~~~~~~~~~~~
+
+The following code shows how you can use the asynchronous
+``DeleteOneAsync()`` method or the synchronous ``DeleteOne()`` method to
+delete one document.
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: delete-one-async
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var result = await _restaurantsCollection.DeleteOneAsync(filter);
+
+   .. tab:: Synchronous
+      :tabid: delete-one-sync
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var result = _restaurantsCollection.DeleteOne(filter);
+
+Delete Multiple Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following code shows how you can use the asynchronous
+``DeleteManyAsync()`` method or the synchronous ``DeleteMany()`` method to
+delete all matched documents.
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: delete-many-async
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var result = await _restaurantsCollection.DeleteManyAsync(filter);
+
+   .. tab:: Synchronous
+      :tabid: delete-many-sync
+
+      .. code-block:: csharp
+         :copyable: true
+
+         var result = _restaurantsCollection.DeleteMany(filter);
+
+Find runnable examples using these methods under :ref:`additional
+information <additional-info>`.
+
+Parameters
+~~~~~~~~~~
+
+The ``DeleteOne()`` and ``DeleteMany()`` methods expect you to pass a
+*query filter* specifying which documents to match. The query filter
+determines which records are selected for deletion based on the criteria
+in :manual:`the query filter document</core/document/#query-filter-documents>`.
+
+Both methods optionally take a ``DeleteOptions`` type as an additional parameter,
+which represents options you can use to configure the delete operation.
+If you don't specify any ``DeleteOptions`` properties, the driver does
+not customize the delete operation.
+
+The ``DeleteOptions`` type allows you to configure options with the
+following properties:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``Collation``
+     - | Gets or sets the type of language collation to use when sorting results.
+
+   * - ``Comment``
+     - | Gets or sets the comment for the operation.
+
+   * - ``Hint``
+     - | Gets or sets the index to use to scan for documents.
+
+   * - ``Let``
+     - | Gets or sets the let document.
```

Comment:
issue: some of these might we worth linking to additional resources, especially collation and let